### ISSUE TITLE
[v1.0.0-beta2] Setting [[graphite]] templates via ENV variable doesn't work

### ISSUE THREAD

**Issue by jackzampolin** [influxdata/influxdb#6943]
### Bug report

**Steps to reproduce:**

```
$ export INFLUXDB_GRAPHITE_0_ENABLED=true
$ export INFLUXDB_GRAPHITE_0_BIND_ADDRESS=":2010"
$ export INFLUXDB_GRAPHITE_0_DATABASE="graphiteTest"

$ export INFLUXDB_GRAPHITE_0_TEMPLATES='["jmeter.test.*.* app..measurement.field"]'
OR
$ export INFLUXDB_GRAPHITE_0_TEMPLATES_0="jmeter.test.*.* app..measurement.field"
OR
$ export INFLUXDB_GRAPHITE_0_TEMPLATES="jmeter.test.*.* app..measurement.field"
OR
$ export INFLUXDB_GRAPHITE_0_TEMPLATES_0='["jmeter.test.*.* app..measurement.field"]'

$ service influxdb start
$ echo "jmeter.test.a.max 23 `date +%s`" | nc -vvv localhost 2010
$ influx
> use graphiteTest
> show measurements
name: measurements
------------------
name
jmeter.test.a.max
```

**Expected behavior:** [What you expected to happen]

Measurement parses properly

**Actual behavior:** [What actually happened]

It failed to

**Comment by miry** [comment]
same issue. And I found that: 

```
$ INFLUXDB_GRAPHITE_0_ENABLED=true INFLUXDB_GRAPHITE_1_ENABLED=true  influxd config
```

return only 1 config.

**Comment by miry** [comment]
```
INFLUXDB_GRAPHITE_1_ENABLED=true  influxd config
```

Generates the default:

```
[[graphite]]
  enabled = false
  bind-address = ":2003"
  database = "graphite"
  retention-policy = ""
  protocol = "tcp"
  batch-size = 5000
  batch-pending = 10
  batch-timeout = "1s"
  consistency-level = "one"
  separator = "."
  udp-read-buffer = 0
```

**Comment by gwossum** [comment]
This will be fixed with #27312. Expected release in InfluxDB 1.13.0.
